### DISCUSSION TITLE
Docs with vertical monitors

### DISCUSSION THREAD

**Discussion by digitaldirk** [microsoft/fluentui-blazor#1837] [Ideas]
Hi,

When referencing the docs I usually have them on a vertical portrait monitor. For these docs the main body content is squished.

I would like the option to collapse the left nav menu and the right aside content (ToC/log) or maybe auto hide/turn into an icon like the mobile layout does.

Don't mind doing a PR, but I think some discussion/owners input would be good first.

Thank you

**Edit:**
Related issue: #1831

**Comment by digitaldirk** [comment]
**Example (1600x900 monitor vertical orientation):**
![image](https://github.com/microsoft/fluentui-blazor/assets/22691956/22f95b7b-f856-4dea-b5a8-b4a634c5b59c)

**Reply by vnbaaij** [reply]
I think we should try to have the minimize menu option available on larger width resolution.  Now the breakpoint is at 768/767 pixels:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/25cfdcd5-e52c-46e7-a49c-30397e564a61)
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/2689ebab-6128-45cb-b7b4-0be464aef32c)
(profile menu issue known...)

Maybe you can work on a PR where it is changed to 900 or 1000px?

**Comment by dvoituron** [comment]
At this moment, the "mobile" view is used when the width < 767px.
We had to make some choices.
It would be possible to add another button (top right) to switch to "mobile" mode at the user's request, but that's not in the roadmap :-)
